server/models/Availability.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const dayOptions = {
+    startTime: {
+      type: Number, 
+      required: [
+        { validator: isDayActive, msg: 'A start time is required if this day is active!'},
+        { validator: isValidTime, msg: 'The start time cannot be after the end time'}
+      ], 
+      default: 360 //6AM
```

Comment:
It looks like we're going with minutes, I would also consider adding `min`/`max` schema validations for start and end time 👍🏽